.github/PULL_REQUEST_TEMPLATE/submission.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Submission
+about: Submit data to the Open Reaction Database
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the submission**
+* High-level description of the data
+* Source information (e.g. DOI)
+* Details on how the dataset was created (e.g. web editor, template enumeration, jupyter, etc.)
+* Details about any issues encoding the data (e.g. missing schema fields, ambiguous data, etc.)
+
+**Attach any supporting files**
+* Reaction template and CSV/Excel for enumerated datasets
```

Comment:
Note: users will likely have to .zip them to attach them

Reply:
Added a note.